prod-devel-rcar-virtio.yaml

```diff
@@ -614,6 +613,9 @@ parameters:
         components:
           dom0:
             builder:
+              conf:
+                - [XT_DOMU_DTB_NAME, "%{XT_DOMU_DTB_NAME}"]
```

Comment:
Why this is needed? There is no dtb for DomU at all AFAIR

Reply:
Thanks, you are right. This is my copy-paste error.